sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -329,17 +329,57 @@ function withAzureCliCredential() {
 }
 ```
 
-#### Authenticating a user account with Visual Studio Code
+#### Authenticating a User Account with Azure PowerShell
```

Comment:
Will we remember adding this on the docs? My memory is too bad to keep track of this. Would it help to make an issue?